jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -141,6 +147,36 @@ def _parse_spectral_cube(app, file_obj, data_type='flux', data_label=None):
     # TODO: SpectralCube does not store data quality information
 
 
+def _parse_spectrum1d_3d(app, file_obj):
+    # Load spectrum1d as a cube
+
+    for attr in ["flux", "mask", "uncertainty"]:
+        if attr == "mask":
+            flux = getattr(file_obj, attr) * file_obj.flux.unit
```

Comment:
I am going to use the `<<` operator here, see https://docs.astropy.org/en/latest/units/index.html#performance-tips .